x-pack/plugins/alerting/server/alerts_service/alerts_service.ts

```diff
@@ -458,4 +459,12 @@ export class AlertsService implements IAlertsService {
       });
     }
   }
+
+  public async setAlertsToUntracked(opts: SetAlertsToUntrackedOpts) {
```

Comment:
I'm unsure that this method actually belongs in `AlertsService` rather than `AlertsClient`, because it interacts with and updates alert data. However, we want to make it available without being connected to a Rule (which is a prerequisite for creating an `AlertsClient`), and it also doesn't actually need a corresponding Rule to operate.

Not sure if there's a specific design goal re: Service vs. Client that I'm violating here with this, but it was the most expedient way I could think of to get this to work.

Reply:
This is probably fine for now. The initial intent was for the `AlertService` to handle all of the framework level resource installation that's required to write alerts for a rule and then `AlertsClient` to be the way for the alerting task runner to interact with those alerts. As you point out, there are use cases for users interacting directly with alerts. In the future, we may want to consolidate all of those interactions into a separate client but this is fine for now.